### PR TITLE
Fix usage of the executeAdminDomainCommand.

### PR DESCRIPTION
**Pull request by mirko-lelansky**
Replace the empty list implementation with a full list implementation to remove
the unsupported operation exceptions.

### CODE REVIEW

File: glassfish-managed-3.1/src/main/java/org/jboss/arquillian/container/glassfish/managed_3_1/GlassFishServerControl.java [hunk 1]
```diff
@@ -43,8 +42,6 @@
             "more information please refer to relevant issues for existing workarounds: https://java.net/jira/browse/GLASSFISH-21004 " +
             "https://issues.apache.org/jira/browse/DERBY-6438";
 
-    private static final List<String> NO_ARGS = Collections.emptyList();
```
Comment (bartoszmajsak) [inline glassfish-managed-3.1/src/main/java/org/jboss/arquillian/container/glassfish/managed_3_1/GlassFishServerControl.java]: Can't we simply change this line to:
```java
private static final List<String> NO_ARGS = new ArrayList<String>();
```

and leave the rest unchanged?

I assume those commands are not changing the state of arguments passed?

Comment (mirko-lelansky) [inline glassfish-managed-3.1/src/main/java/org/jboss/arquillian/container/glassfish/managed_3_1/GlassFishServerControl.java]: Sure this is the finest solution. I will changes this in my branch.

**Comment by s1hofmann** [comment]
Could this patch please be merged into master and an updated version be pushed to Maven central?

**Comment by bartoszmajsak** [comment]
Thanks for your contribution and apologies for pushing it upstream so late. I made minor improvement and landed in ac28d1d.

**Comment by s1hofmann** [comment]
Thank you very much!

**Comment by bartoszmajsak** [comment]
You are welcome.

New version has been released. It will take a bit to land it in central, but normally less than a day. Please note that we are gradually switching to widely known semantic versioning scheme - which essentially means we are dropping `.Final` suffix (we were semantic before too, but we decided to follow well-known convention)

New version is `1.0.1`.

**Pull request closed without merging**